backend/modules/pdf_processing/ai_extraction_service.py

```
@@ -0,0 +1,592 @@
+                debug_log("[WARNING] No JSON found in persona extraction response")
+                raise ValueError("Failed to extract personas: No JSON found in AI response")
+                
+        except Exception as e:
+            debug_log(f"[ERROR] Persona extraction failed: {str(e)}")
+            raise
+    
+    async def generate_scenes(
+        self, 
+        combined_content: str, 
+        title: str, 
+        session_id: Optional[str] = None, 
```

Ruff (0.14.5)

Unused method argument: `session_id`

(ARG002)